addons/docs/src/frameworks/react/__testfixtures__/9586-js-react-memo/docgen.snapshot

```diff
@@ -18,7 +18,7 @@ Button.propTypes = {
   label: PropTypes.string.isRequired,
   onClick: PropTypes.func.isRequired
 };
-const MemoButton = React.memo(Button);
+const MemoButton = /*#__PURE__*/React.memo(Button);
```

Comment:
any idea what's going on here?

Reply:
I see this comment was resolved, but i'll leave this here for anyone else curious https://github.com/babel/babel/pull/11428